text/0015-market-design-revisit.md

```diff
@@ -0,0 +1,116 @@
+
+#### Market Period (14 days)
+
+During the market period, core sales are conducted through a well-established **clearing price Dutch auction** that features a `RESERVE_PRICE`. The price initiates at a premium, designated as `PRICE_PREMIUM` (for instance, 30%) and descends linearly to the `RESERVE_PRICE` throughout the duration of the `MARKET_PERIOD`. Each bidder is expected to submit both their desired price and the quantity (that is, the amount of Coretime) they wish to purchase. To secure these acquisitions, bidders must make a deposit equivalent to their bid multiplied by the chosen quantity, in DOT. 
+
+The market achieves resolution once all quantities have been sold, or the `RESERVE_PRICE` has been reached. This situation leads to determining the `MARKET_PRICE` either by the lowest bid that was successful in clearing the entire market or by the `RESERVE_PRICE`. This mechanism yields a uniform price, shaped by market forces (refer to the following discussion for an explanation of its benefits). In other words, all buyers pay the same price (per unit of Coretime). Further down the benefits of this variant of a Dutch auction is discussed.
+
+**Note:** In cases where some cores remain unsold in the market, all buyers are obligated to pay the `RESERVE_PRICE`.
+
+#### Renewal Period (7 days)
+
+As the `RENEWAL_PERIOD` commences, all current tenants are granted the opportunity to renew their cores at a slight discount of `MARKET_PRICE * RENEWAL_DISCOUNT` (for instance, 10%). This provision affords marginal benefits to existing tenants, balancing out the non-transferability aspect of renewals.
+
+At the end of the period, all available cores are allocated to the current tenants who have opted for renewal and the participants who placed bids during the market period. If the demand for cores exceeds supply, the cores left unclaimed from renewals may be awarded to bidders who placed their bids early in the auction, thereby subtly incentivizing early participation. If the supply exceeds the demand, all unsold cores are transferred to the Instantanous Market.
+
+#### Reserve Price Adjustment
+
+After all cores are allocated, the `RESERVE_PRICE` is adjusted following the process described in RFC-1 and serves as baseline price in the next `BULK_PERIOD`. 
+
+**Note:** The particular price curve is outside the scope of the proposal. The `MARKET_PRICE` (as a function of `RESERVE_PRICE`), however, is able to capture higher demand very well while being capped downwards. That means, the curve that adjusts the `RESERVE_PRICE` should be more sensitive to undercapacity.
+
+#### Settlement Period (7 days)
+
+There are no explicit tasks for the coretime systemchain to perform in this period. This period is still important to allow for sufficient time for all participants to engage in secondary markets, especially to be able to allocate Coretime that starts in the upcoming `BULK_PERIOD`.
```

Comment:
Allocation via secondary markets is something which should be going on _throughout_ the bulk period, not just in advance of it.

The wording doesn't make that clear, and a first reading makes me think it expects all secondary market allocations for the next `BULK_PERIOD` to be settled ahead of time.

Reply:
Yes, the secondary market can function throughout the whole `BULK_PERIOD` trading Coretime that is allocated for that period. In addition, the settlement period allows for a trading in advance of that period. The only restriction that is imposed by this proposal is that we only allocate Coretime after the `MARKET_PERIOD` and `RENEWAL_PERIOD`, because only after we know the allocation of coretime from that `BULK_PERIOD`. Then, secondary market can happen in the settlement period and *throughout* the whole upcoming period.

I'll clarify this.